README.md

```diff
@@ -1,8 +1,8 @@
 # Check Digit Currency Library
 
-The Check Digit currency library is the officially sanctioned method for Check Digit services to deal with currency types, formatting and country/currency relationships. Features:
+The Check Digit currency library is the officially sanctioned method for Check Digit services to deal with currency types, formatting and country/currency relationships at a particular date/time. Features:
 
-- various currency and country lookup functions
+- various currency and country lookup functions at a particular date/time starting '1970-01-01T00:00:00.001Z'
```

Comment:
we're not actually starting at this date however as we're not going back and filling in every country that has existed in the past since 1970. i would just get rid of the 'starting...' part.